pkg/flowkit/config/loader_test.go

```diff
@@ -41,7 +41,10 @@ func Test_JSONSimple(t *testing.T) {
 		},
 		"contracts": {},
 		"networks": {
-			"emulator": "127.0.0.1:3569"
```

Comment:
what is the reason for changing this tests?

Reply:
Reverted the changes for simple config testing since we have test that do multiple network config

Reply:
https://github.com/onflow/flow-cli/pull/449/commits/b94433142876497e2697c8cb40ecc8b135bfaa70